app/javascript/elm/src/TimeRange.elm

```diff
@@ -0,0 +1,56 @@
+module TimeRange exposing (TimeRange(..), defaultTimeRange, update, viewTimeFilter)
+
+import Html exposing (Html, div, h4, input, text)
+import Html.Attributes as Attr
+import Json.Decode as Decode
+import Json.Encode as Encode
+
+
+type TimeRange
+    = TimeRange Range
+
+
+type alias Range =
+    { timeFrom : Int
+    , timeTo : Int
+    }
+
+
+defaultTimeRange : TimeRange
+defaultTimeRange =
+    TimeRange
+        { timeFrom = 1
+        , timeTo = 1
+        }
```

Comment:
This is actually never used cause it's overwritten by flags, I'm thinking of deleting the default model totally.